src/Stratis.Bitcoin.Features.Miner/Controllers/StakingController.cs

```diff
@@ -123,25 +123,31 @@ public IActionResult StartStaking([FromBody]StartStakingRequest request)
         /// <summary>
         /// Stop staking.
         /// </summary>
+        /// <param name="delayInSeconds">Delay, in seconds, after which the staking will stop.</param>
         /// <returns>An <see cref="OkResult"/> object that produces a status code 200 HTTP response.</returns>
         [Route("stopstaking")]
         [HttpPost]
-        public IActionResult StopStaking()
+        public IActionResult StopStakingAsync([FromBody] int delayInSeconds = 0)
```

Comment:
Hey, how come this method has an `async` suffix ? I can't see any 'ayncy' stuff going on in the method.

Reply:
Correct thanks, I changed it but not the name.